scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -18,6 +18,12 @@
 class HttpCompressionMiddleware(object):
     """This middleware allows compressed (gzip, deflate) traffic to be
     sent/received from web sites"""
+
+    def __int__(self, crawler):
+        #setting to decide to keep or discard encoding header
```

Comment:
I think these comments are unnecessary, the setting should be documented instead.